src/beamlime/dashboard/widgets/workflow_status_list_widget.py

```diff
@@ -0,0 +1,276 @@
+from typing import Any
+
+import panel as pn
+
+from beamlime.config.workflow_spec import WorkflowStatus, WorkflowStatusType
+from beamlime.dashboard.workflow_controller_base import WorkflowControllerBase
+
+from .workflow_ui_helper import WorkflowUIHelper
+
+
+class WorkflowStatusUIHelper:
+    """Helper class for workflow status display."""
+
+    @staticmethod
+    def get_status_display_info(status: WorkflowStatus) -> dict[str, str]:
+        """Get display information for a workflow status."""
+        if status.status == WorkflowStatusType.STARTING:
+            return {
+                'color': '#ffc107',  # Yellow
+                'text': 'Starting...',
+                'button_name': 'Stop',
+                'button_type': 'primary',
+                'opacity_style': '',
+            }
+        elif status.status == WorkflowStatusType.RUNNING:
+            return {
+                'color': '#28a745',  # Green
+                'text': 'Running',
+                'button_name': 'Stop',
+                'button_type': 'primary',
+                'opacity_style': '',
+            }
+        elif status.status == WorkflowStatusType.STOPPING:
+            return {
+                'color': '#b87817',  # Orange
+                'text': 'Stopping...',
+                'button_name': 'Stop',
+                'button_type': 'primary',
+                'opacity_style': '',
+            }
+        elif status.status == WorkflowStatusType.STARTUP_ERROR:
+            return {
+                'color': '#dc3545',  # Red
+                'text': 'Error',
+                'button_name': 'Remove',
+                'button_type': 'light',
+                'opacity_style': 'opacity: 0.7;',
+            }
+        elif status.status == WorkflowStatusType.STOPPED:
+            return {
+                'color': '#6c757d',  # Gray
+                'text': 'Stopped',
+                'button_name': 'Remove',
+                'button_type': 'light',
+                'opacity_style': 'opacity: 0.7;',
+            }
+        else:  # UNKNOWN
+            return {
+                'color': '#6c757d',  # Gray
```

Comment:
For later: Is it possible to use CSS to style the dashboard? Scattering styling throughout the Python code makes it difficult to customise and fine tune.